### ISSUE TITLE
Dongle (without voice support) wont connect after running dongle callwaiting

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bg111/asterisk-chan-dongle#132]
```
What steps will reproduce the problem?
1. On a dongle that does not have voice support
2. Run dongle callwaiting (by mistake)
3. Check the state, or just wait on the cli

What is the expected output? What do you see instead?
It should try report failed. And should allow to disable that command. But 
instead I keep getting this error every 15 seconds
[Jun  6 12:07:23] ERROR[20432]: chan_dongle.c:433 do_monitor_phone: [dongle0] 
timedout while waiting 'OK' in response to 'AT'

What version of the product are you using? On what operating system?
Ubuntu 12.04, Asterisk 1.8

Please provide any additional information below.
1) I tried to unplug and plug the device, nothing
2) I tried to reset the dongle, nothing
3) I tried to reload the dongle, nothing
4) I tried "dongle callwaiting disable dongle0", it reported "dongle disabled". 
Basically a catch 22 on this one.
5) I tried to commend out the following configuration lines from dongle.conf 
autodeletesms=yes, usecallingpres=yes, callingpres=allowed_passed_screen, 
callwaiting=auto
6) I tried to change value of callwaiting to yes, no 

None of the above got me rid of my mistake.
```

Original issue reported on code.google.com by `jsiddha...@mcruiseon.com` on 6 Jun 2013 at 6:40
